src/js/components/TextArea/TextArea.js

```diff
@@ -49,6 +61,7 @@ const TextArea = forwardRef(
           focus={focus}
           value={value}
           focusIndicator={focusIndicator}
+          style={{ height: `${height}px` }}
```

Comment:
If I use my mouse to adjust the TextArea size and make the height larger than the original size, then press the 'down' arrow, the TextArea jumps back to a smaller size